Formula/valkyrie.rb

```diff
@@ -14,8 +14,14 @@ class Valkyrie < Formula
   depends_on "valgrind"
 
   def install
+    inreplace "src/utils/vk_utils.h", "#include <iostream>",
+                                      "#include <iostream>\n#include <unistd.h>"
```

Comment:
They've already been told they need to include that file for Solaris, but I'm happy to pile on.